test/runBTagAnalyzer_cfg.py

```diff
@@ -243,14 +243,46 @@
     VarParsing.multiplicity.singleton,
     VarParsing.varType.string,
     'JP Calibration pyload to use')
-options.register('storeCSVTagVariables', True,
+options.register('storeJetVariables', True,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to store Jet Variables')
+options.register('storeQuarkVariables', False,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to store c/b quark Variables')
+options.register('storeHadronVariables', False,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to store Hadron Variables')
+options.register('storeGenVariables', False,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to store Gen Variables')
+options.register('storeCSVTagVariables', False,
     VarParsing.multiplicity.singleton,
     VarParsing.varType.bool,
     'True if you want to keep CSV TaggingVariables')
-options.register('storeDeepFlavourTagVariables', True,
+options.register('storeDeepFlavourTagVariables', False,
     VarParsing.multiplicity.singleton,
     VarParsing.varType.bool,
     'True if you want to keep DeepFlavour TaggingVariables')
+options.register('storeDeepFlavourVariables', True,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to keep DeepFlavour Jet Variables')
+options.register('storeDeepCSVVariables', True,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to keep DeepCSV Jet Variables')
+options.register('storePFElectronVariables', False,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to keep PF Electron Variables')
+options.register('storePFMuonVariables', False,
+    VarParsing.multiplicity.singleton,
+    VarParsing.varType.bool,
+    'True if you want to keep DeepCSV Jet Variables')
```

Comment:
In the future we should add either a stringy option that allows to set "commissioning" "ttSF" etc. options all at once, or create dedicated defaults and chain them, e.g.: ```defaults=Moriond18,Commissioning```